src/Service/Slugify.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
+ * Auteur: Khaled Benharrat, Damien Sarrazy, Kevin Chalumeau
+ * Date: 30/06/2020
+ */
+
+//src/Service/Slugify.php
+namespace App\Service;
+
+class Slugify
+{
+    protected $input;
```

Comment:
This property seems never used so useless.

Reply:
Ok, this variable is not used.